apps/studio/components/layouts/TableEditorLayout/EntityListItem.tsx

```diff
@@ -333,7 +394,18 @@ const EntityListItem: ItemRenderer<Entity, EntityListItemProps> = ({
               }}
             >
               <Download size={12} />
-              <span>Export as CSV</span>
+              <span>Export table as CSV</span>
+            </DropdownMenuItem>
+            <DropdownMenuItem
```

Comment:
i'd opt to do a secondary dropdown menu for this - otherwise the dropdown menu here seems to be getting larger and clunkier

Reply:
hah yes of course, thanks!